ExtLibs/ArduPilot/parampck.cs

```diff
@@ -97,6 +100,16 @@ public static MAVLink.MAVLinkParamList unpack(byte[] data)
                 Array.Copy(data, dataPointer, vdata, 0, mapped.type_len);
                 dataPointer += mapped.type_len;
 
+                double default_value = Double.NaN;

```

Comment:
can we change this to "decimal" from double.  the reason being a recent rounding issue, and decimal internally deals with this better.

Reply:
changed to a nullable decimal `decimal?`

Reply:
sure. makes sence